<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Задачи</string>
  </property>
  <widget class="QListWidget" name="tasks_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="load_tasks_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>381</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Загрузить задачи</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="task_title">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Название задачи</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>276</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Описание</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="task_description">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_task_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>381</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить задачу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_task_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>381</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить задачу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_task_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>381</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить задачу</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
